application/client/src/app/ui/tabs/sources/multiplefiles/file.holder.ts

```diff
@@ -0,0 +1,98 @@
+import { Matcher } from '@matcher/matcher';
+import { Matchee } from '@module/matcher';
+import { bytesToStr } from '@env/str';
+import { FileType } from '@platform/types/files';
+
+export function detectFileType(filename: string) {
+    const ext: string | undefined = filename.split('.').pop();
+    switch (ext) {
+        case 'dlt':
+            return FileType.Dlt;
+        case 'pcap':
+        case 'pcapng':
+            return FileType.Pcap;
+        default:
+            return FileType.Text;
+    }
+}
+
+export interface IFileInfo {
+    filename: string;
+    modificationDate: number;
+    name: string;
+    path: string;
+    size: number;
+    type: FileType | string;
+}
```

Comment:
I am not sure, why you need one more type definition for file?
`application/platform/types/files.ts: File` looks like has everything what you need:
```
export interface File {
    name: string; //name of file
    ext: string; //extention
    path: string; //path
    filename: string; //full file name
    type: FileType; //type of file (DLT, pcap, text)
    stat: Stat; 
}
```